app/routes/meetings/components/MeetingDetail.js

```diff
@@ -116,96 +124,104 @@ class MeetingDetails extends Component<Props> {
     const canEdit = actionGrant && actionGrant.includes('edit');
 
     return (
-      <div className={styles.root}>
-        {showAnswer && (
-          <h2>
-            {' '}
-            Du har nå svart på invitasjonen{' '}
-            <span aria-label="smile" role="img">
-              😃
-            </span>{' '}
-          </h2>
-        )}
-
-        <FlexRow className={styles.heading}>
-          <div style={{ flex: 1 }}>
-            <NavigationTab title={meeting.title} className={styles.detailTitle}>
-              <NavigationLink to="/meetings">
-                <i className="fa fa-angle-left" /> Mine møter
-              </NavigationLink>
-              {canEdit && (
-                <NavigationLink to={`/meetings/${meeting.id}/edit`}>
-                  Endre møte
+      <div>
+        <Content>
+          <ContentHeader className={styles.heading}>
+            <div style={{ flex: 1 }}>
+              <NavigationTab
+                title={meeting.title}
+                className={styles.detailTitle}
+              >
+                <NavigationLink to="/meetings">
+                  <i className="fa fa-angle-left" /> Mine møter
                 </NavigationLink>
-              )}
-              {canDelete && (
-                /* $FlowFixMe what is wrong with confirmomdalwithparent */
-                <ConfirmModalWithParent
-                  title="Slett møte"
-                  message="Er du sikker på at du vil slette møtet?"
-                  onConfirm={this.onDeleteMeeting}
-                >
-                  <div>
-                    <NavigationLink to="">Slett møte</NavigationLink>
-                  </div>
-                </ConfirmModalWithParent>
-              )}
-            </NavigationTab>
-            <h3>
-              <Time
-                style={{ color: 'grey' }}
-                time={meeting.startTime}
-                format="ll [-] HH:mm"
-              />
-            </h3>
-          </div>
-        </FlexRow>
-        <div className={styles.mainContent}>
-          <FlexItem className={styles.reportContent} flex={2}>
-            <h2>Referat</h2>
-            <DisplayContent content={meeting.report} />
-          </FlexItem>
-          <FlexItem className={styles.statusContent} flex={1}>
-            <Card style={{ border: 'none', padding: 0 }} shadow={false}>
-              <ul>
-                {this.attendanceButtons(statusMe, meeting.startTime)}
-                {statusMe && (
+                {canEdit && (
+                  <NavigationLink to={`/meetings/${meeting.id}/edit`}>
+                    Endre møte
+                  </NavigationLink>
+                )}
+                {canDelete && (
+                  /* $FlowFixMe what is wrong with confirmomdalwithparent */
```

Comment:
You should be able to remove this now, since we fixed a the flow error in the `ConfirmModalWithParent` component.

Reply:
Removed the comment if that was all.

Reply:
Cannot remove it without getting this error message.
![screenshot from 2017-11-17 13-19-09](https://user-images.githubusercontent.com/25204035/32946969-ff712f8e-cb99-11e7-9406-0550b04d38b4.png)

Reply:
Ahh. I guesd its a problem with the component then.. I can look at it, so just leave it here for now